package.json

```diff
@@ -15,6 +15,7 @@
     "cors": "^2.8.5",
     "custom-env": "^2.0.2",
     "express": "^4.19.2",
+    "jsonwebtoken": "^9.0.2",
     "mongoose": "^8.3.2",
```

Comment:
Why adding jwt? this pr should be related to changing the schema only

Reply:
sorry, will remove. forgot I added that. I accidently did git add ./, and forgot I changed that